pom.xml

```diff
@@ -175,4 +190,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-</project>  
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <configuration>
+                    <compatibleSinceVersion>2.0</compatibleSinceVersion>
```

Comment:
What specifically is incompatible? I will need to have a list for release notes. It seems like you are updating everything relevant in `readResolve` so what am I missing?